molecule/answerfile_str/molecule.yml

```diff
@@ -0,0 +1,40 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: rhv_he_deploy
+    image: centos:7
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  options:
+    v: true
+    # skip the step that actually runs hosted-engine,
+    # this scenario only tests answerfile generation
+    skip-tags: rhv_he_deploy_run
+scenario:
+  name: answerfile_str
+  # idempotence fails because of random tempdir answerfile location
```

Comment:
Is this still true given the check in the code?

Reply:
The check relies on the output of `hosted-engine --check-deploy` switching from 1 to 0 once hosted-engine is deployed. Since the scenario doesn't actually deploy hosted-engine (and, in fact, never installs it), the check will never cause the tasks to bypass the answerfile templating. 

An idempotent tmpdir would fix this, but probably isn't worth it. It might be worth noting somewhere that the role itself is idempotent unless you enable log collection; it's only these scenarios that are not.

Reply:
And after reading what I just wrote even if the idempotent tmpdir was fixed, the non-idempotent log collection (because the logs themselves change) means that these scenarios can probably never be idempotent, since the depend on log collection to get at the answerfile for parsing.

Reply:
Is there a way we can ask Molecule to ignore the idempotence of a single task? Or can we skip the non-idempotent task(s or at least set their changed_when status) in a certain environment?